csvkit/utilities/csvstat.py

```diff
@@ -114,6 +114,13 @@ def add_arguments(self):
         self.argparser.add_argument(
             '--count', dest='count_only', action='store_true',
             help='Only output total row count.')
+        self.argparser.add_argument(
+            '--decimal-format', dest='decimal_format', type=str,
+            help='%%-format specification for printing decimal numbers. '
+                 'Defaults to locale-specific formatting with \'%%.3f\'.')
+        self.argparser.add_argument(
+            '--thousands-grouping', dest='thousands_grouping',
```

Comment:
Let's use "grouping separator" since some locales have groups of 2, 4, etc.

Reply:
Also, shouldn't this be `--no-grouping-separator`, since the current behavior is to include it, and we typically don't change current behavior unless there's a bug?

Reply:
The reason I wanted to change the default is that having thousands separators by default breaks parsing in most libraries that parse numbers. (I gave an example with `bc` in the related issue.)

Sure it's nicer to read as a human, but for quickly summarizing data and piping output, it's not very usable.

Reply:
Hmm, `csvstat`'s output is already not very machine-readable. And in general CSV Kit is designed to be human-friendly by default. I don't see an issue with having to add `--no-grouping-separator` to get machine-readable output.

Reply:
I see. Maybe I'm (ab)using the csvstat tool. It's just that it's so useful in combination with the other tools!

Your suggestion is a good alternative. I will change the PR accordingly.